lambda-kinesis-lambda/README.md

```diff
@@ -0,0 +1,70 @@
+# A demonstration of serverless producer-consumer use case using AWS Lambda and AWS Kinesis Data Stream
+
+This pattern creates an AWS Kinesis Data Stream, a stream consumer, and an AWS Lambda function. When data is added to the stream, the Lambda function is invoked.
```

Comment:
This pattern creates 2 lambda functions, one that puts data in the stream and another that gets trigger when there is data in the stream. Please improve this description, to what the code is doing

Reply:
Added as per suggestion